pkg/injector/vault/injector.go

```diff
@@ -360,14 +352,7 @@ func (i *SecretInjector) InjectSecretsFromVaultPath(paths string, inject SecretI
 			if !i.config.IgnoreMissingSecrets {
 				return errors.Errorf("path not found: %s", valuePath)
 			}
-
-			i.logger.Warn(
-				fmt.Sprintf(
-					"path not found - We couldn't find a secret path. This is not an error since missing secrets can be ignored according to the configuration you've set (annotation: %s).",
-					common.VaultIgnoreMissingSecretsAnnotation,
```

Comment:
we can keep this message, but change to
```
"path not found - We couldn't find a secret path. This is not an error since missing secrets can be ignored according to the configuration you've set"
```